data/card_catalogue/unique-1.json

```diff
@@ -204,7 +251,10 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Pick a tile up to 2 tiles away. The next charcter to walk on this tile will fall over.",
+      "rules": {
+        "active": "Pick a point on the floor up to 2 metres away. The next NPC to walk there will fall over.",
```

Comment:
Don't fancy forcing characters to fall down, this is the sort of place where I'd have them do an easy save just for laughs if they forgot it was there. 
Presently thinking this card should summon something so we could put it on the map. Will open an issue for that later.

Reply:
- #113